### PR TITLE
Pract.05 | Add 04-10 | Nikola Svetoslavov

### PR DESCRIPTION
**Pull request by Justsvetoslavov**

### CODE REVIEW

**Comment by GeorgiTerziev02** [comment]
Your pr is not following the rules. Please provide description! @Justsvetoslavov 🤡

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

File: SI/Sem.04/Pract.04/Task04.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿int abs(int num1, int num2)
+{
+    return ((num1 - num2) >= 0 ? (num1 - num2) : (num2 - num1));
```
[the remark this answers is not in the document]
Comment (Justsvetoslavov) [reply SI/Sem.04/Pract.04/Task04.cpp:3]: True,

I tend to keep it due to different compilers.

File: SI/Sem.04/Pract.04/Task06.cpp [hunk 2]
```diff
@@ -0,0 +1,4 @@
+﻿bool isCapitalLetter(char S)
+{
+	return (S >= 'A' && S <= 'Z');
```
[the remark this answers is not in the document]
Comment (Justsvetoslavov) [reply SI/Sem.04/Pract.04/Task06.cpp:3]: Will remove these.

File: SI/Sem.04/Pract.04/Task07.cpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+{
+	return num >= interval_lower_bound && num <= interval_upper_bound
+}
+
+long long askUser(long long from, long long to)
+{
+	long long number;
+	while (std::cin >> number)
+	{
+		if (isNumberInInterval(number,from,to)) {
```
[the remark this answers is not in the document]
Comment (Justsvetoslavov) [reply SI/Sem.04/Pract.04/Task07.cpp:11]: VSCode does not respect it :(

File: SI/Sem.04/Pract.04/Task09.cpp [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+	{
+		case '+': return a + b;
+		
+		case '-': return a - b;
+		
+		case '*': return a * b;
+		
+		case '/': return a / b;
+		
+		case '%': return a % b;
```
[the remark this answers is not in the document]
Comment (Justsvetoslavov) [reply SI/Sem.04/Pract.04/Task09.cpp:13]: Was thinking whether this check is not appropriate before passing to function.

However, I agree to add it.

**Review by GeorgiTerziev02 — APPROVED** [review]

**Pull request merged**